src/MUSI6106Exec/MUSI6106Exec.cpp

```diff
@@ -55,6 +86,11 @@ int main(int argc, char* argv[])
         return -1;
     }
     phAudioFile->getFileSpec(stFileSpec);
+    
+    CCombFilter::create(phFilterFile);
+    phFilterFile->init(CCombFilter::kCombFIR, fDelay, stFileSpec.fSampleRateInHz, stFileSpec.iNumChannels);
```

Comment:
This init function nothing, but looking at the arguments, I would expect it to create an instance of a kCombFIR filter, set the delay value for the filter to fDelay*samplerate (in samples), and process iNumChannels

Reply:
This init function does nothing*